### PR TITLE
Improve typing

### PR DESCRIPTION
**Pull request by haeminn**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts [hunk 1]
```diff
@@ -448,17 +445,17 @@ export class AreaChartNormalizedComponent extends BaseChartComponent {
     return opts;
   }
 
-  updateYAxisWidth({ width }): void {
+  updateYAxisWidth({ width }: { width: number }): void {
```
Comment (Hypercubed) [inline projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts:448]: These look odd to me... are we passing some object that includes `width`?

**Review by Hypercubed — COMMENTED** [review]
One comment.

Hard to tell from the PR but are we also exporting all the new enums?

File: projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts (hunk 1 above)
Comment (haeminn) [reply projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts:448]: Yeah, it seems like it.

**Comment by haeminn** [comment]
> One comment.
> 
> Hard to tell from the PR but are we also exporting all the new enums?

@Hypercubed  Yes
https://github.com/swimlane/ngx-charts/blob/367bccd8595946bd04616260bbd95a67c085ca40/projects/swimlane/ngx-charts/src/public-api.ts#L49

https://github.com/swimlane/ngx-charts/blob/367bccd8595946bd04616260bbd95a67c085ca40/projects/swimlane/ngx-charts/src/public-api.ts#L26

**Review by Hypercubed — APPROVED** [review]

**Pull request merged**